roles/etcd/tasks/etcdctl.yml

```diff
@@ -4,9 +4,9 @@
   when: not openshift.common.is_atomic | bool
 
 - name: Configure etcd profile.d alises
-  copy:
-    src: etcdctl.sh
-    dest: /etc/profile.d/etcdctl.sh
+  template:
+    dest: "/etc/profile.d/etcdctl.sh"
```

Comment:
@ingvagabund Were there problems with the static file? this change makes the template depend on a lot of variables which may not be set when running via playbooks/common/openshift-cluster/upgrades/etcd/main.yml